### PR TITLE
Fixes #21640: Uprading relay to 7.2 fails on ubuntu 22

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/21640

### CODE REVIEW

**Review by amousset — APPROVED** [review]

File: rudder-relay/debian/postinst [hunk 1]
```diff
@@ -51,6 +51,11 @@ case "$1" in
         mv /opt/rudder/etc/rudder-pkg/rudder-pkg.conf.distrib /opt/rudder/etc/rudder-pkg/rudder-pkg.conf
       fi
 
+      # sometimes (ubuntu22 ?) divert prevents the file to be present
+      if [ ! -f /opt/rudder/etc/rudder-networks-24.conf ]; then
+        touch /opt/rudder/etc/rudder-networks-24.conf
```
Comment (amousset) [inline rudder-relay/debian/postinst:56]: shouldn't we move the .distrib file?

Comment (peckpeck) [reply rudder-relay/debian/postinst:56]: we shouldn't need to, but we can, this way the directory would be cleaned from .distrib files

**Comment by peckpeck** [comment]
PR updated with a new commit

File: rudder-relay/debian/postinst [hunk 2]
```diff
@@ -51,6 +51,14 @@ case "$1" in
         mv /opt/rudder/etc/rudder-pkg/rudder-pkg.conf.distrib /opt/rudder/etc/rudder-pkg/rudder-pkg.conf
       fi
 
+      # sometimes (ubuntu22 ?) divert prevents the file to be present
+      if [ ! -f /opt/rudder/etc/rudder-networks-24.conf.distrib ]; then
```
Comment (amousset) [inline rudder-relay/debian/postinst:55]:
```suggestion
      if [ -f /opt/rudder/etc/rudder-networks-24.conf.distrib ]; then
```

**Pull request merged**